text/003-theme.md

```
@@ -223,207 +245,392 @@ const [themeClass, vars] = theme({
+  get casualGreeting() {
+    console.log(this.otherInfo.city);
+    // Thanks to ThisType, this here is inferred to be TTarget & GreetingMixin
+    return this.greeting(this.name, this.age);
+  }
+});
+
+console.log(personWithGreeting.casualGreeting); // "Hello, I'm John and I'm 30 years old!"
+```
+
+# Drawbacks
+[drawbacks]: #drawbacks
```

markdownlint-cli2 (0.18.1)

Link and image reference definitions should be needed
Unused link or image reference definition: "drawbacks"

(MD053, link-image-reference-definitions)